content/en/docs/collector/deployment/centralized.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Centralized
+description:
+  Why and how to send signals to a single OTLP end-point and from there to
+  backends
+weight: 3
+---
+
+The centralized collector deployment pattern consists of applications (or other
```

Comment:
I’d have two main scenarios with separate pictures. First one with multiple applications sending to one Collector (basic setup). The second one could be this load-balanced scenario, but with multiple applications to visualize the centralized pattern.